tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -234,7 +234,7 @@ def get_brick_details(self, objects, integration_id):
             for _brick in _bricks:
                 resource_detail = {}
                 resource_detail["host_name"] = _cnc.fqdn.replace(".", "_")
-                if _brick.deleted in ['true', 'True', 'TRUE']:
+                if str(_brick.deleted) in ['true', 'True', 'TRUE']:
```

Comment:
Why are you complicating the code?

A simple
`str(_brick.deleted).lower == 'true'`   is enough

Reply:
i will change this